apps/marketplace/components/SellerAssessment/SellerAssessmentCriteriaStage.scss

```diff
@@ -1,4 +1,9 @@
 .criteria label {
   margin-top: 1.5rem;
   width: 100%;
-}
\ No newline at end of file
+}
+
+p.essential {
+  color: #313131;
```

Comment:
This may not be needed.  Is the essential text assigned this colour by other styles if you remove this?

Reply:
I had a look at the other css files like main.css and they don't have that specific colour (#313131) that I can assign the text to.